master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -124,6 +125,11 @@ public void updateWorkerLostMeta(
     workerLostEvents.remove(worker);
   }
 
+  public void updateWorkerExcludeMeta(WorkerInfo worker) {
```

Comment:
Since WorkerLost will remove worker from excludedWorkers, If we add excludeWorkers by workerExcludeEvents, then if worker comes back, excludedWorkers  would not contain these workers. This cause inconsistent state with workerExcludeEvents.